### ISSUE TITLE
router: disabling IPv6 due to system IPv6 config: open /proc/sys/net/ipv6/conf/all/disable_policy: no such file or directory

### ISSUE THREAD

**Issue by bradfitz** [tailscale/tailscale#1241]
I have working IPv6:

```
root@dev:~#  lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 10 (buster)
Release:	10
Codename:	buster

root@dev:~# ping6 google.com
PING google.com(sea15s11-in-x0e.1e100.net (2607:f8b0:400a:808::200e)) 56 data bytes
64 bytes from sea15s11-in-x0e.1e100.net (2607:f8b0:400a:808::200e): icmp_seq=1 ttl=117 time=4.23 ms
64 bytes from sea15s11-in-x0e.1e100.net (2607:f8b0:400a:808::200e): icmp_seq=2 ttl=117 time=14.3 ms
64 bytes from sea15s11-in-x0e.1e100.net (2607:f8b0:400a:808::200e): icmp_seq=3 ttl=117 time=28.4 ms
```

But with tailscale 1.4.1:

```
Jan 31 18:53:49 dev tailscaled[30686]: Creating wireguard device...
Jan 31 18:53:49 dev tailscaled[30686]: Creating router...
Jan 31 18:53:49 dev tailscaled[30686]: router: disabling IPv6 due to system IPv6 config: open /proc/sys/net/ipv6/conf/all/disable_policy: no such file or directory
Jan 31 18:53:49 dev tailscaled[30686]: router: dns: using dns.directManager
Jan 31 18:53:49 dev tailscaled[30686]: Bringing wireguard device up...
Jan 31 18:53:49 dev tailscaled[30686]: UDP bind has been updated
```

Bogus check?

```
root@dev:~# cat /proc/sys/net/ipv6/conf/all/disable_ipv6 
0
root@dev:~# cat /proc/sys/net/ipv6/conf/all/disable_policy
cat: /proc/sys/net/ipv6/conf/all/disable_policy: No such file or directory
```

/cc @danderson @josharian

**Comment by bradfitz** [comment]
From chat, some of my Debian buster machines have that file and some don't (slightly different kernel patch version).

And even on the machine above without it, it still works:

```
root@dev:~# ip -6 rule add lookup main
root@dev:~# ip -6 rule add lookup main
root@dev:~# ip -6 rule add lookup main
root@dev:~# ip -6 rule add lookup main
root@dev:~# ip -6 rule show
0:	from all lookup local 
32761:	from all lookup main 
32762:	from all lookup main 
32763:	from all lookup main 
32764:	from all lookup main 
32765:	from all lookup main 
32766:	from all lookup main
```

**Comment by danderson** [comment]
TIL, disable_policy is not always present on machines that support policy routing, so its absence should be a "maybe?" not a firm "no". PR out to fix that, we'll get it into a 1.4 patch release.
